scripts/load_tester/start.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+import sys
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+TX_SERVICE_URL = os.getenv("TRANSACTION_SERVICE_URL")
+PRINT_FORMAT = "{0:<10} {1:>8}::{2:>8}"
+
+
+def check_service():
+    local_instance_check = requests.get("http://localhost:8000/about")
+    if local_instance_check.status_code != 200:
+        print("Local instance of the service must be running")
+        sys.exit(-1)
+
+
+def load_safes() -> list[str]:
+    print("Loading remote ...")
+    top_safes_url = TX_SERVICE_URL + "/api/v1/analytics/multisig-transactions/by-safe/?limit=300"
+    print(top_safes_url)
+    response = requests.get(top_safes_url)
+    return list(map(lambda safe: safe['safe'], response.json()['results']))
+
+
+check_service()
+safes = load_safes()
+print("Top 300 safes:")
+print("\n\t" + "\n\t".join(safes))
+print("Safes ready ... ")
+print("Populating cache ...")
+
+for safe in safes:
+    balance_response = requests.get("http://localhost:8000/v1/safes/" + safe + "/balances/USD")
```

Comment:
As the purpose is to populate our **staging** cache we shouldnot use localhost by default

Reply:
I thought it was meant for local testing. I will change this.

Reply:
I will add more logic to determine the environment, as the top 300 safes are fetched from the `.env` core services url

Reply:
meaning I will `sys.exit` if you try to fetch from production envs

Reply:
I would not limit it to any specific env, but the default should be staging (at the minimum it should not be hardcoded)

Reply:
> As the purpose is to populate our staging cache we shouldnot use localhost by default

I thought it was just for staging. I will leave it to work on any network or environment.